src/filter.js

```diff
@@ -257,50 +257,50 @@ define(function (require) {
     }
   };
 
-  /**
-   *  Constructor: <code>new p5.LowPass()</code> Filter.
-   *  This is the same as creating a p5.Filter and then calling
-   *  its method <code>setType('lowpass')</code>.
-   *  See p5.Filter for methods.
-   *
-   *  @class p5.LowPass
-   *  @constructor
-   *  @extends p5.Filter
-   */
-  p5.LowPass = function() {
-    p5.Filter.call(this, 'lowpass');
-  };
-  p5.LowPass.prototype = Object.create(p5.Filter.prototype);
+  // /**
+  //  *  Constructor: <code>new p5.LowPass()</code> Filter.
+  //  *  This is the same as creating a p5.Filter and then calling
+  //  *  its method <code>setType('lowpass')</code>.
+  //  *  See p5.Filter for methods.
+  //  *
+  //  *  @class p5.LowPass
+  //  *  @constructor
+  //  *  @extends p5.Filter
+  //  */
+  // p5.LowPass = function() {
+  //   p5.Filter.call(this, 'lowpass');
+  // };
+  // p5.LowPass.prototype = Object.create(p5.Filter.prototype);
```

Comment:
@therewasaguy do we want to leave these classes commented out or do we just need to add `@for` to the jsdoc comments?